src/ToData.purs

```diff
@@ -1,38 +1,138 @@
 module ToData
   ( class ToData
+  , class ToDataArgs
+  , class ToDataWithIndex
+  , class ToDataArgsRL
+  , genericToData
+  , toDataArgsRec
   , toData
+  , toDataArgs
+  , toDataWithIndex
   ) where
 
 import Prelude
 
+import ConstrIndex (class HasConstrIndex, constrIndex)
+import Data.Array (cons)
 import Data.Array as Array
 import Data.BigInt (BigInt)
 import Data.BigInt as BigInt
 import Data.Either (Either(Left, Right))
 import Data.Foldable (class Foldable)
+import Data.Generic.Rep as G
 import Data.List (List)
 import Data.Map (Map)
 import Data.Map as Map
 import Data.Maybe (Maybe(Just, Nothing))
 import Data.Profunctor.Strong ((***))
 import Data.Ratio (Ratio, denominator, numerator)
+import Data.Symbol (class IsSymbol, SProxy(..), reflectSymbol)
 import Data.Tuple (Tuple(Tuple))
-import Data.Tuple.Nested (type (/\))
+import Data.Tuple.Nested (type (/\), (/\))
```

Comment:
Redundant import `(/\)`

Reply:
Sorry about this, I need to find a purs linter -.-